prow/config/config.go

```diff
@@ -1070,6 +1071,44 @@ func loadConfig(prowConfig, jobConfig string) (*Config, error) {
 	if err := yamlToConfig(prowConfig, &nc); err != nil {
 		return nil, err
 	}
+	for _, additionalProwConfigDirs := range additionalProwConfigDirs {
+		var errs []error
+		errs = append(errs, filepath.Walk(additionalProwConfigDirs, func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				// Finish walking and handle all errors in bulk at the end, otherwise this is annoying as a user
+				errs = append(errs, err)
+				return nil
+			}
+			if strings.HasPrefix(info.Name(), "..") {
+				// kubernetes volumes also include files we
+				// should not look be looking into for keys
```

Comment:
Not quite sure I understand this comment

Reply:
I've extended the comment, is it sufficent?